RELEASING.md

```diff
@@ -90,21 +85,22 @@ After releasing a new major or minor version (e.g. 1.9.0), a maintenance branch
 1.9.1) would then be done within that branch and any development for the next
 major or minor release can continue in master.
 
-After creating a maintenance branch, the `extra.branch-alias.dev-master` field
-in the master branch's `composer.json` file should be updated. For example,
-after branching v1.9, `composer.json` in the master branch may still read:
+When work begins on a major new version, create a maintenance branch for the
+last minor version and update the `extra.branch-alias.dev-master` field
+in the master branch's `composer.json` file. For example, after branching v1.99,
+`composer.json` in the master branch may still read:
 
 ```
 "branch-alias": {
-    "dev-master": "1.9.x-dev"
+    "dev-master": "1.x-dev"
```

Comment:
You won't need branch-alias in v2 if you decide to delete the master branch and use a branch named 2.x or 2.0.